lib/android/src/androidTest/java/net/skyscanner/backpack/reactnative/dialog/DialogViewManagerTest.kt

```diff
@@ -105,24 +107,31 @@ class DialogViewManagerTest {
   @Test
   fun test_icon() {
     val view = manager.createViewInstance(themedContext)
+    val skyBlueValue = ContextCompat.getColor(themedContext, R.color.bpkSkyBlue)
     manager.updateProperties(view, buildProps(
       "icon", JavaOnlyMap.of(
         "iconId", "bpk_food",
         "iconColor", "bpkSkyBlue")))
 
     assertNotNull(view.state.icon)
-    assertEquals(BpkDialog.Icon(R.drawable.bpk_food, R.color.bpkSkyBlue), view.state.icon)
+    assertEquals(BpkDialog.Icon(R.drawable.bpk_food, skyBlueValue), view.state.icon)
   }
 
   @Test
-  fun test_icon_invalid_values() {
+  fun test_icon_invalid_id() {
     val view = manager.createViewInstance(themedContext)
     assertThat({
       manager.updateProperties(view, buildProps(
         "icon", JavaOnlyMap.of(
         "iconId", "invalid",
         "iconColor", "bpkSkyBlue")))
     }, throws(android.content.res.Resources.NotFoundException::class))
+  }
+
+  @Test
+  @Ignore
```

Comment:
Is it possible to add a comment here as to why we have ignored this? Just so people coming across this code have understanding as to reasons